README.md

```diff
@@ -114,6 +114,33 @@ A value which has a Setoid must provide an `equals` method. The
 
 2. `equals` must return a boolean (`true` or `false`).
 
+### Ord
+
+A value that implements the Ord specification must also implement
+the [Setoid](#setoid) specification.
+
+1. `a.compare(a) <= 0` (reflexivity)
+2. If `a.compare(b) <= 0` and `b.compare(a) <= 0`, then `a.equals(b) === true` (antisymmetry)
+3. If `a.compare(b) <= 0` and `b.compare(c) <= 0`, then `a.compare(c) <= 0` (transitivity)
+
+#### `compare` method
+
+```hs
+compare :: Ord a => a ~> a -> Ordering
+```
+
+A value which has an Ord must provide a `compare` method. The
+`compare` method takes one argument:
+
+     a.compare(b)
+
+1. `b` must be a value of the same Ord
+
+    1. If `b` is not the same Ord, behaviour of `compare` is
+       unspecified (returning NaN is recommended).
+
+2. `compare` must return a number (-1, 0 or 1).
```

Comment:
I suggest backticks around `-1`, `0`, and `1` (and `NaN`). Also, an Oxford comma. ;)